frigg/builds/models.py

```diff
@@ -316,6 +316,18 @@ def initiate_deployment(self, options):
         deployment.start()
         return deployment
 
+    def delete_logs(self):
+        """
+        This will replace all log with the default content
+        """
+        result = self.result
```

Comment:
I don't think that will work since result is a OneToOneField and update is a method on querysets.